apps/DisplayCluster/resources/MasterContentWindow.qml

```diff
@@ -119,21 +164,24 @@ BaseContentWindow {
             id: fixedButtonsDelegate
             Column {
                 CloseControlButton {
-                    TouchMouseArea {
+                    MultiTouchGestureArea
+                    {
                         anchors.fill: parent
-                        onTap: closeWindow()
+                        onTapStarted: closeWindow()
```

Comment:
should be onTapEnded at least, even though it's potentially not the same as the old onTap